README.md

```diff
@@ -18,6 +18,11 @@ Learn more at the [Sycamore website](https://sycamore-rs.netlify.app), or stop b
 Issue reports and Pull Requests are always welcome!
 Check out the [section on contributing](https://sycamore-rs.netlify.app/contribute/architecture) in the docs.
 
+## MSRV
+
+The minimum supported Rust toolchain is v1.53.0,
+due to usage of or-patterns syntax.

```

Comment:
```suggestion
The minimum supported Rust toolchain is v1.53.0. Sycamore is not guaranteed to compile on an older version of Rust.
```

Reply:
Bumping MSRV is an explicit (and somewhat rare) thing, and one that does it would know why, and would not miss updating this. An alternative is to only care about supporting latest `stable`, and that's when I think the reason is not needed.

Reply:
Sorry, I wasn't being clear here. What I meant was that or-patterns might not be the only 1.53 feature we use. I seem to recall that `RefCell::take` and a few other functions were also stabilized recently.

Reply:
The only error I get when compiling with 1.52 is about or-patterns.

Reply:
But setting our MSRV to 1.53.0 does not mean that or-patterns is the only feature we are allowed to use. And technically, we could remove or-patterns from the code base to make it compatible with older versions of Rust. It's not like it's as crucial as some other features like `specialization` or `fn_traits`.